Controllers/UserinfoController.cs

```diff
@@ -56,4 +56,28 @@ public class UserinfoController : ControllerBase
 
         return CreatedAtAction(nameof(GetUserInterests), new GetTechInterestDto(persistedInterests.Entity));
     }
+
+    [Authorize]
+    [HttpPut("interests")]
+    public async Task<IActionResult> UpdateInterest(
+        [FromServices] IKwizDbContext dbContext,
+        [FromBody] IEnumerable<string> userSelectedInterests,
+        CancellationToken cancellationToken = default)
+    {
+        if (userSelectedInterests.Any() is false)
+            return BadRequest("User must select at least one interest.");
+
+        var idCliam = HttpContext.User.FindFirstValue(ClaimTypes.NameIdentifier);
+        var userId = Guid.Parse(idCliam);
+        var interests = await dbContext.TechInterests
+            .FirstOrDefaultAsync(t => t.UserId == userId, cancellationToken);
+
```

Comment:
if(interests is null)
    BadRequest("User has never selected interests before.");